bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -9712,33 +9768,9 @@ public TagName addOrUpdateTagName(String displayName, String description, TagNam
 	 *
 	 * @throws TskCoreException
 	 */
+	@Deprecated
```

Comment:
Please add an @deprecated entry to the method header comment as well, wherever we are deprecating existing methods.